lib/main.rb

```diff
@@ -0,0 +1,11 @@
+def substrings(text, dictionary)
+  matches = {}
+
+  dictionary.each do |key|
+    matches_count = text.scan(/(#{key})/ix).size
```

Comment:
why `/ix`? The `x` ignores whitespaces, so this will result in a match if there are letters that are part of separate words. For example "below ertrude" will match with "lower"

Reply:
Oh! I thought it evaluates each word!! Understood!

Reply:
Done!